sdk/identity/Azure.Identity/src/ChainedTokenCredential.cs

```diff
@@ -88,18 +88,17 @@ private async ValueTask<AccessToken> GetTokenImplAsync(bool async, TokenRequestC
                         groupScopeHandler.Dispose(default, default);
                         return token;
                     }
-                    catch (AuthenticationFailedException e)
+                    catch (CredentialUnavailableException e)
                     {
                         exceptions.Add(e);
                     }
-                    catch (Exception e) when (!(e is OperationCanceledException))
+                    catch (Exception e)
```

Comment:
The filter `when (!(e is OperationCanceledException))` is meant to prevent cancellation exceptions from being wrapped. What's the rational behind removing it?

Reply:
Fixed